sidedish/src/component/ItemDetail/Counter.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IoChevronUp, IoChevronDown } from 'react-icons/io5';
+
+const Counter = ({ count, handleClickUp, handleClickDown }) => {
+  return (
+    <StyledCounter>
+      <div className="counter">{count}</div>
+      <div className="arrow_container">
+        <IoChevronUp className="arrow" onClick={handleClickUp}></IoChevronUp>
+        <IoChevronDown className="arrow" onClick={handleClickDown}></IoChevronDown>
```

Comment:
```suggestion
        <IoChevronDown className="arrow" onClick={handleClickDown} / >
```
컴포넌트에 children이 없을 경우 self-closing tag를 사용하시면 가독성이 좋긴 합니다 ㅎㅎ
https://betterprogramming.pub/21-best-practices-for-a-clean-react-project-df788a682fb

Reply:
앗 children 이 없는데 selfClose를 안한 부분이 있었군요 ㅜㅜ 감사합니다!!